apps/minifront/src/components/layout.tsx

```diff
@@ -35,7 +35,7 @@ export const Layout = () => {
   return (
     <>
       <HeadTag />
-      <div className='flex min-h-screen w-screen flex-col'>
+      <div className='flex min-h-screen w-full flex-col'>
```

Comment:
This is the actual fix. `w-screen` was setting the div's width to `100vw`, which causes overflow when there's a _vertical_ scrollbar. `w-full` sets it to `100%`, which accounts for the vertical scrollbar.

The reason we only saw this problem on the dashboard, is that the dashboard was the only page with a vertical scrollbar!